bitrise.yml

```diff
@@ -500,6 +500,8 @@ workflows:
 
             echo "Swiftlint version before mirror swap"
             swiftlint version
+            # Workaround to find the homebre-core folder
+            brew tap homebrew/core --force
```

Comment:
This makes the trick, however it takes longer to run this step ~1.5 mins... 

Wondering if we need this complete step... @adudenamedruby  as the swiftlint master

Reply:
Uh..... I've never needed to do this so I'm not sure. This would be a homebrew thing. This seems fine, though I'm no brew expert, but if it's working then, yay!